### ISSUE TITLE
SMTP server config no longer working in `anvil-app-server` 1.5.x

### ISSUE THREAD

**Issue by samcarswell** [anvil-works/anvil-runtime#39]
We have a yaml file with the following SMTP options:
```yaml
smtp-host: example.net
smtp-port: 465
smtp-encryption: ssl
smtp-username: admin@example.net
smtp-password: example_password
```

This works fine for `anvil-app-server` 1.4, however using 1.5.x produces the following the error:

```python
InternalError: No SMTP server has been configured for this app
Traceback:
  /opt/bitnami/python/lib/python3.7/site-packages/anvil_downlink_worker/anvil/_threaded_server.py:404
  /opt/bitnami/python/lib/python3.7/site-packages/anvil_downlink_worker/anvil/server.py:40
  /opt/bitnami/python/lib/python3.7/site-packages/anvil_downlink_worker/anvil/email.py:37
  AusTrakka/UserManagementServer.py:241
  AusTrakka/UserManagementServer.py:377
  app/AusTrakka/Utils/Users/UserAdmin.py:266
  app/AusTrakka/Utils/Users/UserLogin.py:54
  app/AusTrakka/HomeForm/__init__.py:91
```

Passing these as CLI options produces the same result.

**Comment by meredydd** [comment]
Hi @samcarswell,

I believe this is now fixed as of 1.5.5. Can you confirm?

**Comment by samcarswell** [comment]
Hi @meredydd, yes I can confirm that this is now fixed in `1.5.5`. Thank you!
